packages/ui/src/components/MessageInput/index.tsx

```diff
@@ -3,6 +3,8 @@ import { useState } from 'react';
 import { TextArea } from '../../core';
 import { MessageInputContainer, MessageInputProps } from './MessageInputBase';
 
+export const DEFAULT_MESSAGE_INPUT_HEIGHT = 44;
```

Comment:
Can we actually apply this to the input to enforce it matching?

Reply:
We do, in /MessageInput/index.native.tsx

Reply:
Ahh I see -- can index.native import the constant from this file then? Nbd if not though.

Reply:
Now using the single definition.

Reply:
Causes a require cycle, reverting for now.

Reply:
Dang, thanks for giving it a try.

Reply:
I'm fixing the keyboardavoidingview issue and it'll obviate the need for this anyway